components/gallery/GalleryItem.vue

```diff
@@ -268,4 +295,37 @@ $break-point-width: 930px;
 .h-audio {
   height: 70%;
 }
+
+@import '@/styles/abstracts/variables';
```

Comment:
```suggestion
```

Reply:
> Suggested change
	@import '@/styles/abstracts/variables';

not sure why delete this import? it used `theme` in the following 🤔️

Reply:
> > Suggested change
> > @import '@/styles/abstracts/variables';
> 
> not sure why delete this import? it used `theme` in the following 🤔️

If you look at line 233 on this file, you would find that It's a duplicate import